### ISSUE TITLE
Duration check in random_subsegment 

### ISSUE THREAD

**Issue by yinruiqing** [pyannote/pyannote-generators#6]
If the segment duration shorter than the random_subsegment duration. It will cause a problem.
Code
```python
from pyannote.generators.fragment import random_subsegment
print(next(random_subsegment(Segment(10,20),30)))
```
Result:
`
[-00:00:09.772 -->  00:00:20.227]
`

**Comment by hbredin** [comment]
Thanks for reporting this. 
What should be the expected behavior? 
Should we raise a `ValueError`?

**Comment by hbredin** [comment]
Will be fixed in next release (see e990f19)
